server/src/main/resources/application-local.yml

```diff
@@ -1,5 +1,23 @@
+micronaut:
+  security:
+    enabled: true
+    reject-not-found: false
```

Comment:
This is so we return 404 instead of 401/403 when using SECURED("SOME_ROLE") at the Controller level